.rustfmt.toml

```diff
@@ -9,7 +9,6 @@ format_code_in_doc_comments = false
 comment_width = 80
 normalize_comments = true # changed
 normalize_doc_attributes = false
-license_template_path = "FILE_TEMPLATE" # changed
```

Comment:
I had to disable this, because this feature is kind of [broken](https://github.com/rust-lang/rustfmt/pull/4967). I kept getting `license check failed` from rustfmt even after doing `cat FILE_TEMPLATE src/client.rs > /tmp/f && mv /tmp/f src/client.rs`